src/containers/ViewerPanel/index.tsx

```diff
@@ -127,7 +127,6 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
     }
 
     public componentDidMount() {
```

Comment:
but doesn't this mean there is still erroneously an old error in state? Even though we aren't showing it?

Reply:
That is a good point. I was thinking that whenever a viewer loads a new trajectory it does clear out the error, but it just happens after `componentDidMount`. That is true, but it doesn't account for when you leave the viewer with errors and return to the viewer without loading a new trajectory. I'm about to push up another commit